store/getters.js

```diff
@@ -639,6 +642,12 @@ export const latestFamily = (state) => {
   return state.families ? state.families[state.families.length - 1] : null;
 };
 
+export const hasFFPEandWGtag = (sample) => {
```

Comment:
```suggestion
export const isNotFFPEandWGtag = (sample) => {
```
I wonder if it would make sense to name it something along the lines of what WG actually stands for, to make it more into an abstraction. what do you think?

Reply:
Something like isNotFFPEinWholeGenomeTag ?